### PR TITLE
feat: added map icon in location activity

### LINKED ISSUE #1475: Adding a maps icon to the Location Activity
**Actual Behaviour**

Currently when viewing the locations user has no way of viewing the location on the map.

**Expected Behaviour**

Adding a maps icon on the toolbar and enabling the user to locate the places with the help of it.

**Steps to reproduce it**

-

**LogCat for the issue**

-

**Screenshots of the issue**

Where-ever possible add a screenshot of the issue.

**Would you like to work on the issue?**

Yes

Milestone:

https://github.com/fossasia/open-event-android/milestone/24

### PR DESCRIPTION
**Pull request by arp95**
Fixes #1475 

Changes: 
Added maps icon to location acitvity so that user can directly view the location on the maps.

Screenshots for the change: 

18378710_10211225239821284_1110483670_o

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 1]
```diff
@@ -35,6 +39,8 @@
 public class LocationActivity extends BaseActivity implements SearchView.OnQueryTextListener {
     final private String SEARCH = "searchText";
 
+    private static final String FRAGMENT_TAG_REST = "fgtr";
```
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: Better naming please

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 2]
```diff
@@ -139,14 +145,33 @@ protected void onDestroy() {
...
+                FragmentManager fragmentManager = getSupportFragmentManager();
+                FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+                fragmentTransaction.replace(R.id.content_frame_location,
+                        ((OpenEventApp) getApplication())
+                                .getMapModuleFactory()
+                                .provideMapModule()
+                                .provideMapFragment(), FRAGMENT_TAG_REST).commit();
+                return true;
+            
+            default:
```
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: What is the default case?

**Comment by arp95** [comment]
@mananwason 
made the required changes.
please review!

**Comment by arp95** [comment]
@mananwason @iamareebjamal 
any changes i need to do?

**Comment by iamareebjamal** [comment]
No. Please let the Travis build finish, then I'll merge

**Comment by arp95** [comment]
@iamareebjamal 
sure. didnt see that.

**Comment by iamareebjamal** [comment]
I guess @mananwason should review the changes as he requested them.

**Comment by mananwason** [comment]
@iamareebjamal @arp95 You need to be patient with the reviews. @iamareebjamal I was going to review it tonight. Please let @the-dagger or @mariobehling or @championswimmer or me review it before merging. You can definitely approve it and then the mentor can take it forward from there and merge it.

**Comment by iamareebjamal** [comment]
Ok. Sorry for that @mananwason

Will keep in mind from next time on

**Comment by mananwason** [comment]
No problems @iamareebjamal 🙂

**Comment by Shailesh351** [comment]
@arp95 Map fragment is hiding behind tracks and back button on action bar is not responding

**Comment by mananwason** [comment]
@Shailesh351 Can you open a new issue and add screenshots over there. Also reference this PR

**Comment by arp95** [comment]
@mananwason
actually i discussed with @iamareebjamal and i have already opened an issue to resolve it. the disappeareance of map icon is causing problems which i am working on.

**Comment by mananwason** [comment]
@arp95 @iamareebjamal Please test the code before making a PR or merging other PR's. It will be really difficult to track what went wrong with so many people on board and lots of PR's being merged.

**Comment by iamareebjamal** [comment]
@mananwason A new issue was created as it was unrelated to this PR and also occurring in master branch code in other Activity. Nevertheless, I now wait for mentors to merge PRs

**Pull request merged**